assets/js/modules/ads/components/notifications/AdsModuleSetupCTABanner.js

```diff
@@ -0,0 +1,235 @@
+	const onSetupCallback = useCallback( () => {
+		if (
+			! shouldShowWooCommerceRedirectModal ||
+			isWooCommerceRedirectModalDismissed
+		) {
+			markNotificationDismissed();
+			activateModule();
+			setIsSaving( true );
+			return;
+		}
+
+		setOpenDialog( true );
+	}, [
+		shouldShowWooCommerceRedirectModal,
+		activateModule,
+		markNotificationDismissed,
+		isWooCommerceRedirectModalDismissed,
+	] );
+
+	const onModalDismiss = useCallback( () => {
+		markNotificationDismissed();
+		setOpenDialog( false );
+	}, [ markNotificationDismissed, setOpenDialog ] );
+
+	const showTooltip = useShowTooltip( id );
+	const { isTooltipVisible } = useTooltipState( id );
+
+	if ( isTooltipVisible ) {
+		return (
+			<AdminMenuTooltip
+				title=""
+				content={ __(
+					'You can always enable Ads from Settings later',
+					'google-site-kit'
+				) }
+				dismissLabel={ __( 'Got it', 'google-site-kit' ) }
+				tooltipStateKey={ id }
+			/>
+		);
+	}
+
+	// TODO: Don't use `skipHidingFromQueue` and remove the need to check
+	// if this component should output anything.
+	//
+	// We "incorrectly" pass true to the `skipHidingFromQueue` option when dismissing this banner.
+	// This is because we don't want the component removed from the DOM as we have to still render
+	// the `AdminMenuTooltip` in this component. This means that we have to rely on manually
+	// checking for the dismissal state here.
+	if ( hideCTABanner ) {
+		return null;
+	}
+
+	return (
+		<Notification>
+			<NotificationWithSVG
+				id={ id }
+				title={ __(
+					'Get better quality leads and enhance conversions with Ads',
+					'google-site-kit'
+				) }
+				description={
+					<Description
+						text={ __(
+							'Help drive sales, leads, or site traffic by getting your business in front of people who are actively searching Google for products or services you offer.',
+							'google-site-kit'
+						) }
+						learnMoreLink={
+							<LearnMoreLink
+								id={ id }
+								label={ __( 'Learn more', 'google-site-kit' ) }
+								url={ learnMoreURL }
+							/>
+						}
+					>
+						{ isAdBlockerActive && (
+							<AdBlockerWarning moduleSlug="ads" />
+						) }
+					</Description>
+				}
+				actions={
+					<ActionsCTALinkDismiss
+						id={ id }
+						className="googlesitekit-setup-cta-banner__actions-wrapper"
+						ctaLabel={ __( 'Set up Ads', 'google-site-kit' ) }
+						onCTAClick={ onSetupCallback }
+						dismissOnCTAClick={ false }
+						isSaving={ isSaving }
+						dismissLabel={
+							isDismissalFinal
+								? __( 'Don’t show again', 'google-site-kit' )
+								: __( 'Maybe later', 'google-site-kit' )
+						}
+						dismissOptions={ {
+							skipHidingFromQueue: true,
+						} }
+						onDismiss={ showTooltip }
+						dismissExpires={ 2 * WEEK_IN_SECONDS }
+					/>
+				}
+				SVG={ breakpointSVGMap[ breakpoint ] || AdsSetupSVG }
+			/>
+			{ openDialog && (
+				<WooCommerceRedirectModal
+					dialogActive
+					onDismiss={ onModalDismiss }
+				/>
```

Comment:
Properties with no values should go at the end:
```suggestion
				<WooCommerceRedirectModal
					onDismiss={ onModalDismiss }
					dialogActive
				/>
```